pipeline/lib/util.ml

```diff
@@ -6,3 +6,14 @@ let get_branch_name branch =
   let prefix = "refs/heads/" in
   let len_prefix = String.length prefix in
   String.sub branch len_prefix (String.length branch - len_prefix)
+
+let current_timestamp () =
```

Comment:
`current_timestamp` seems a bit misleading. Should we rename this to something like `stale_commit_timestamp`?